From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Martijn Muijsers <martijnmuijsers@live.nl>
Date: Wed, 23 Nov 2022 21:03:04 +0100
Subject: [PATCH] Gale configuration

License: MIT (https://opensource.org/licenses/MIT)
Gale - https://galemc.org

This patch is based on the following patch:
"Timings v2"
By: Aikar <aikar@aikar.co>
As part of: Paper (https://github.com/PaperMC/Paper)
Licensed under: MIT (https://opensource.org/licenses/MIT)

diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index 9bab00ab10c78908090c8a1a12d4c84e9324b08b..544791edbd13d4ee711d22e6c8ef48329944e99c 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -2385,6 +2385,21 @@ public interface Server extends PluginMessageRecipient, net.kyori.adventure.audi
         {
             throw new UnsupportedOperationException("Not supported yet.");
         }
+
+        // Gale start - Gale configuration - API
+        /**
+         * @deprecated Server config options may be renamed or removed without notice. Another API will be provided
+         * once the APIs of other configurations change.
+         *
+         * @return The server's gale config.
+         */
+        @Deprecated(since = "1.21.4", forRemoval = true)
+        @NotNull
+        public org.bukkit.configuration.file.YamlConfiguration getGaleConfig() {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+        // Gale end - Gale configuration - API
+
         // Paper end
 
         /**
